.vite/deps_temp_6dcee04d/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "type": "module"
+}
```

Comment:
I don't think the `.vite` folder should be committed to git.  I'm not even sure why it's here :thinking: